contrib/ansible/roles/skydive_dev/tasks/lxd.yml

```diff
@@ -0,0 +1,27 @@
+---
+- selinux:
+    policy: targeted
+    state: permissive
+
+- shell: "dnf copr enable -y ganto/lxd"
```

Comment:
this is not going to work on centos

Reply:
The COPR repository only supports Fedora. I'm disabling LXD for CentOS